hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/EnrichmentProcessor.java

```diff
@@ -105,6 +107,12 @@ public void process(StructuredTrace trace) {
   }
 
   private void applyEnricher(Enricher enricher, StructuredTrace trace) {
+    Map<String, AttributeValue> attributeMap = trace.getAttributes().getAttributeMap();
+    AttributeValue dropTraceAttrValue = attributeMap.get(DROP_TRACE_ATTRIBUTE);
+    boolean shouldDropTrace = dropTraceAttrValue != null;
```

Comment:
No need to check the value of `dropTraceAttrValue`, just the presence of it fine. The place it is used prior, does it have meaning with true/false?

Reply:
Ah! just checked, just presence is fine.